java/src/test/java/ai/rapids/cudf/TestUtils.java

```diff
@@ -211,4 +214,16 @@ private static boolean hasInf(int v) {
     });
     return result;
   }
+
+  public static File getResourceAsFile(String resourceName) {
+    URL url = TableTest.class.getClassLoader().getResource(resourceName);
```

Comment:
nit: is it important to to emphasize TableTest.class? can this be `TestUtils.class.getClassLoader.getResource(...)` to keep it self-contained.

Reply:
Good catch.  This was a leftover from before I refactored the method.